.rubocop.yml

```diff
@@ -4,10 +4,17 @@
 # https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
 # https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml
 
-Rails:
-  Enabled: true
-
-Layout/AlignParameters:
+AllCops:
+  Exclude:
+    - 'node_modules/**/*'
+    - 'public/**/*'
+  TargetRubyVersion: 3.0.4
+  UseCache: true
+  DisabledByDefault: true
```

Comment:
My approach to these changes is to bring more currency to Rubocop and the gems; without breaking any functionality. This in preparation for a forthcoming addition for inherited proofing.

Regarding the sharing of a .rubocop.yml, not sure it makes (total) sense here; the primary reason being, this is a Sinatra app, so I thought removing the Rails cop was good; not completely sure about that. Otherwise, sure sharing the same cops may make sense; but how? (https://github.com/percy/percy-style)

Reply:
> My approach to these changes is to bring more currency to Rubocop and the gems; without breaking any functionality. This in preparation for a forthcoming addition for inherited proofing.

But changing the `DisabledByDefault` would be a breaking change, since we'd no longer be enforcing any of the rules which we were previously enabled by default, right?

> Regarding the sharing of a .rubocop.yml, not sure it makes (total) sense here; the primary reason being, this is a Sinatra app, so I thought removing the Rails cop was good;

Ah, I had a brain fart and overlooked the fact that this is not a Rails application. That makes sense.

>Otherwise, sure sharing the same cops may make sense; but how? (https://github.com/percy/percy-style)

Yeah, not totally sure about this in the Ruby/Rubocop landscape. For JavaScript, we have a [shared ESLint plugin](http://npmjs.com/package/@18f/eslint-plugin-identity) that's published to NPM, and it includes optional support for different configurations based on the project's needs (which could be a good way to separate out the Rails-specific configuration, if a similar convention would exist in a published Gem configuration).